src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/RecallTest.kt

```diff
@@ -92,70 +92,91 @@ class RecallTest {
+    val recall = recall.copy(
+      inCustodyAtBooking = true,
+      assessedByUserId = ::UserId.random().value,
+    )
+
+    assertThat(recall.status(), equalTo(Status.AWAITING_DOSSIER_CREATION))
+  }
+
+  @Test
+  fun `NOT In Custody Recall with assessedByUserId and assessedByUserId set returns status ASSESSED`() {
+    val recall = recall.copy(
+      inCustodyAtBooking = false,
+      inCustodyAtAssessment = false,
+      assessedByUserId = ::UserId.random().value,
+    )
+
+    assertThat(recall.status(), equalTo(Status.ASSESSED_NOT_IN_CUSTODY))
+  }
+
+  @Test
+  fun `NOT in Custody Recall with bookedByUserId, assessedByUserId & warrantReferenceNumber set returns status AWAITING_RETURN_TO_CUSTODY`() {
     val recall = recall.copy(
       inCustodyAtBooking = false,
       inCustodyAtAssessment = false,
       bookedByUserId = ::UserId.random().value,
-      recallNotificationEmailSentDateTime = OffsetDateTime.now(),
+      assessedByUserId = ::UserId.random().value,
       warrantReferenceNumber = WarrantReferenceNumber(randomString())
     )
 
     assertThat(recall.status(), equalTo(Status.AWAITING_RETURN_TO_CUSTODY))
   }
 
   @Test
-  fun `In Custody Recall with bookedByUserId, recallNotificationEmailSentDateTime ignores warrantReferenceNumber and returns status RECALL_NOTIFICATION_ISSUED`() {
+  fun `In Custody Recall with bookedByUserId, assessedByUserId ignores warrantReferenceNumber and returns status AWAITING_DOSSIER_CREATION`() {
     val recall = recall.copy(
       inCustodyAtBooking = true,
       bookedByUserId = ::UserId.random().value,
-      recallNotificationEmailSentDateTime = OffsetDateTime.now(),
+      assessedByUserId = ::UserId.random().value,
       warrantReferenceNumber = WarrantReferenceNumber(randomString())
     )
 
-    assertThat(recall.status(), equalTo(Status.RECALL_NOTIFICATION_ISSUED))
+    assertThat(recall.status(), equalTo(Status.AWAITING_DOSSIER_CREATION))
   }
 
   @Test
-  fun `In Custody Recall with bookedByUserId and recallNotificationEmailSentDateTime but no assignee returns status RECALL_NOTIFICATION_ISSUED`() {
+  fun `In Custody Recall with bookedByUserId and assessedByUserId but no assignee returns status AWAITING_DOSSIER_CREATION`() {
     val recall = recall.copy(
       inCustodyAtBooking = true,
       bookedByUserId = ::UserId.random().value,
-      recallNotificationEmailSentDateTime = OffsetDateTime.now()
+      assessedByUserId = ::UserId.random().value,
     )
 
-    assertThat(recall.status(), equalTo(Status.RECALL_NOTIFICATION_ISSUED))
+    assertThat(recall.status(), equalTo(Status.AWAITING_DOSSIER_CREATION))
   }
 
   @Test
-  fun `NOT in Custody Recall with bookedByUserId, recallNotificationEmailSentDateTime & assignee set returns status RECALL_NOTIFICATION_ISSUED`() {
+  fun `NOT in Custody Recall with bookedByUserId, assessedByUserId & assignee set returns status ASSESSED_NOT_IN_CUSTODY`() {
```

Comment:
Covered by previous, new, test with return of `ASSESSED_NOT_IN_CUSTODY ` hence delete as duplicate?

Reply:
subtly different as includes assignee?

Reply:
Tho assignee no longer considered for NIC, so deleted